x-pack/solutions/security/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/hooks.tsx

```diff
@@ -207,14 +210,21 @@ const useTypesFilterInitialState = ({
 
   // v8.13 onwards
   // for showing agent types and action types in the same filter
-  if (isSentinelOneV1Enabled || isCrowdstrikeEnabled) {
+  if (isSentinelOneV1Enabled || isCrowdstrikeEnabled || isMicrosoftDefenderEnabled) {
     if (!isFlyout) {
       return [
         {
           label: FILTER_NAMES.agentTypes,
           isGroupLabel: true,
         },
-        ...RESPONSE_ACTION_AGENT_TYPE.map((type) =>
+        ...RESPONSE_ACTION_AGENT_TYPE.filter((agentType) => {
+          switch (agentType) {
+            case 'microsoft_defender_endpoint':
```

Comment:
Q: why do we need the check here?

Reply:
because the feature flag is not yet enabled - see line below (223)

Reply:
makes sense, thanks!